src/app/app.component.ts

```diff
@@ -10,6 +10,9 @@ import { AppState } from './app.service';
  * Top Level Component
  */
 @Component({
+  host:{
+    class:'app app-component flex-container in-column-direction flex-grow-1'
```

Comment:
I'm not sure why you want to do it this way, but it you do, the word "class" needs to be in quotes.